addons/BLUAlert/BLUAlert.lua

```diff
@@ -76,6 +73,7 @@ function get_action_id(targets)
     end
 end
 
+-- Register the action event listener
```

Comment:
This comment doesn't really add much info that isn't immediately obvious from the next line - I would remove it.